packages/quantic/force-app/main/default/lwc/quanticModal/quanticModal.js

```diff
@@ -0,0 +1,60 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticModal` component provides the logic to easily implement a modal in Quantic. This component handles the animation logic, exposes methods to open and close the modal, offers the option to open the modal in full screen or just to cover the search interface, and exposes a set of slots to fully customize the modal content.
```

Comment:
I see "modal" is used as a noun in this file. But since it is an adjective, it should be followed by a noun (e.g., modal container, modal box, modal window).

I personally like `modal container`.

@jpmarceau Do you think of a better term?

Reply:
Actually in the context of front-end development, it's OK to use 'modal' as a noun too :) 
Examples:
https://blog.hubspot.com/website/modal-web-design
https://www.w3schools.com/w3css/w3css_modal.asp

Reply:
Ah! Good to know. Thank you!

Reply:
"QuanticX component provides the logic to easily implement a X in Quantic" i feel like this could apply to any component so I'm not sure this first sentence is very useful.
What about "The `QuanticModal` is a container component that displays slotted content in a modal."

Reply:
That sounds like a good idea to me 👍